### DISCUSSION TITLE
graphic wierdness

### DISCUSSION THREAD

**Discussion by michaelsarr** [picocomputer/community#37] [Q&A] [answered]
i was playing around with trying to work out some simple functions to draw stuff
im seeing some distortion in the middle of the screen
is anyone else experiencing this ?
![IMG_6114](https://github.com/picocomputer/discussions/assets/73266612/2b78b021-8f16-4eb0-8aa6-59982d631940)
![IMG_6116](https://github.com/picocomputer/discussions/assets/73266612/be98a8c2-db62-41e4-af64-74f13356ec51)
![IMG_6115](https://github.com/picocomputer/discussions/assets/73266612/2087b5e8-3dfd-494a-b8cc-eceead39fb28)

code is at
https://github.com/michaelsarr/graphics_library/tree/main

the distortion goes away on a power cycle

**Comment by rumbledethumps** [comment]
What monitor and cables are you using? What VGA mode is the Pi Pico set to?

**Reply by michaelsarr** [reply]
the monitor and cable are pretty old, I think i was seeing this at home too (different monitor and cable)
i can certainly test some more

MODE 1

**Reply by michaelsarr** [reply, marked as answer]
cinching down the cables seemed to do the trick, my bad. move along nothing to see here.
